docs/src/app/(public)/(content)/react/utils/use-render/demos/render-callback/css-modules/index.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+import * as React from 'react';
+import { useRender } from '@base-ui-components/react/use-render';
+import styles from './index.module.css';
+
+type CounterState = {
+  odd: boolean;
+};
+
+type CounterProps = {
+  className?: string;
+  render?: useRender.RenderProp<CounterState>;
+};
+
+function Counter(props: CounterProps) {
+  const { render = <button type="button" />, className, ...otherProps } = props;
+  const [count, setCount] = React.useState(0);
+  const odd = count % 2 === 1;
+  const state = React.useMemo(() => ({ odd }), [odd]);
+
+  const { renderElement } = useRender({
+    render,
+    state,
+    props: {
+      ...otherProps,
+      className: `${styles.Button} ${className ?? ''}`,
+      type: 'button',
```

Comment:
We're setting `type: 'button'` here but also in the props destructuring above. Can we leave just one? Ideally this one.